Project.toml

```diff
@@ -58,6 +58,6 @@ SciMLBase = "2"
 StaticArrays = "1"
 StatsBase = "0.34"
 SurfaceFluxes = "0.11, 0.12"
-Thermodynamics = "0.12.4"
+Thermodynamics = "0.12.8"
```

Comment:
why does land require v0.12.8 now?

Reply:
I wanted to make sure that in the Coupler all packages can run with Thermodynamics v 0.12.8. Will it not cause problems that we set the patch version here?

Reply:
If we specify 0.12.4 here, land will be compatible with anything in the range [0.12.4, 0.13.0). Since our `test/` Manifest isn't checked in, our tests run with the most recent allowed version of each package (so 0.12.8 for Thermodynamics), so that will tell us if land is able to run with Thermodynamics v0.12.8. We only want to update the patch compat in land if land requires something introduced in 0.12.8 (e.g. a bugfix) to be able to run